### PR TITLE
rename package co to cocoyaxi

### PR DESCRIPTION
**Pull request by idealvin**
Specify library name and version:  **cocoyaxi/v2.0.3**

- Rename package co to [cocoyaxi](https://github.com/idealvin/cocoyaxi).
- Add cocoyaxi v2.0.3
---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by idealvin** [comment]
Could someone help? Thanks!
```sh
-- Found cocoyaxi: 2.0.3 (found version "2.0.3") 
-- Library co found /home/conan/w/BuildSingleReference/.conan/data/cocoyaxi/2.0.3/_/_/package/053ea29eb0edc6b1695c893b738a971110c756fd/lib/libco.a
-- Skipping already existing target: CONAN_LIB::cocoyaxi_co
-- Found: /home/conan/w/BuildSingleReference/.conan/data/cocoyaxi/2.0.3/_/_/package/053ea29eb0edc6b1695c893b738a971110c756fd/lib/libco.a
-- Configuring done
-- Generating done
CMake Error at CMakeLists.txt:10 (add_executable):
  Target "test_package" links to target "cocoyaxi::co" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
```

File: recipes/cocoyaxi/all/conanfile.py [hunk 1]
```diff
@@ -89,8 +89,8 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = ["co"]
-        self.cpp_info.names["cmake_find_package"] = "co"
-        self.cpp_info.names["cmake_find_package_multi"] = "co"
+        self.cpp_info.names["cmake_find_package"] = "cocoyaxi"
+        self.cpp_info.names["cmake_find_package_multi"] = "cocoyaxi"
```
Comment (SpaceIm) [inline recipes/cocoyaxi/all/conanfile.py:93]:
```suggestion
```

your library doesn't export CMake config files, so do not suggest in `package_info()` that it does. But I would recommend to add this feature in your CMakeLists, it's very useful for consumers.

File: recipes/cocoyaxi/all/test_package/CMakeLists.txt [hunk 2]
```diff
@@ -5,7 +5,7 @@ set(CMAKE_CXX_STANDARD 11)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
 
-find_package(co REQUIRED)
+find_package(cocoyaxi REQUIRED)
 
 add_executable(${PROJECT_NAME} test_package.cc)
-target_link_libraries(${PROJECT_NAME} co::co)
+target_link_libraries(${PROJECT_NAME} cocoyaxi::co)
```
Comment (SpaceIm) [inline recipes/cocoyaxi/all/test_package/CMakeLists.txt:11]:
```suggestion
target_link_libraries(${PROJECT_NAME} cocoyaxi::cocoyaxi)
```

**Review by SpaceIm — CHANGES_REQUESTED** [review]

**Comment by SpaceIm** [comment]
You should open 2 PRs:
- one PR which add deprecated = "cocoyaxi" attribute to `co` recipe
- one PR adding `cocoyaxi` recipe

**Comment by idealvin** [comment]
> * deprecated

I'm sorry. But I don't know how to make a package deprecated in Conan.

**Comment by SpaceIm** [comment]
> I'm sorry. But I don't know how to deprecate a recipe in Conan.

see my comment above, it's just one line.

**Comment by idealvin** [comment]
> > I'm sorry. But I don't know how to deprecate a recipe in Conan.
> 
> see my comment above, it's just one line.

@SpaceIm  ok, thanks. Another question, should I remove old versions in cocoyaxi?

**Pull request closed without merging**